src/components/GameView/GameCard.vue

```diff
@@ -288,6 +308,38 @@ export default {
   transform: translateY(-32px);
 }
 
+.threesPlayer{
+  animation: slidePlayerThree 2s ease;
+}
+.threesOpponent{
+  animation: slideOpponentThree 2s ease;
+}
+
+@keyframes slidePlayerThree {
+  100% {
+    transform: translate(200px, 50px);
+    opacity: 0;
+  }
+}
+@keyframes slidePlayerThreePortrait {
+  100% {
+    transform: translateY(200px);
+    opacity: 0;
+  }
+}
+@keyframes slideOpponentThree {
+  100% {
+    transform: translate(200px, -200px);
+    opacity: 0;
+  }
+}
+@keyframes slideOpponentThreePortrait {
+  100% {
+    transform: translateY(-200px);
+    opacity: 0;
+  }
+}
```

Comment:
These look great, though they're a bit inconsistent with how we've done the transitions for the other cards. For the rest of them, we've used `<transition>` elements which basically apply certain classes when elements leave and certain styles when they enter. Then we essentially style the end states and give the elements `transition: all <some_duration>` and let the transition system do the rest. 

That said, this is pretty concise and I bet `<transition>` would take more code, so I'm not positive it's actually better to use the transition tag for this use case. @seriouslysean penny for your thoughts.

One thing I will say is we've named most of the transition classes after the directions they move things in, rather than the specific card they apply to. e.g. `.in-below-out-left` or `.in-above-out-below`. 

But that was with the intention of reusing those styles, which we might not actually do here since this somewhat specifically moves stuff from the scrap pile towards the hands. But if we did, we could maybe call them `out-up-and-left` and `out-down-and-right` or something. 

Again I'm not actually sure what's best here. It's interesting that this is different than the approach seen in the `<GameView>` but it's also a specific use case that may not be reused and it's clear and concise. Hmm!

Reply:
I think switching it to a transition element makes sense, to keep everything in order.